pkg/controller/pravegacluster/upgrade_test.go

```diff
@@ -752,9 +752,9 @@ var _ = Describe("Pravega Cluster Version Sync", func() {
 					_ = client.Get(context.TODO(), req.NamespacedName, foundPravega)
 				})
 
-				It("should set rollback condition reason to UpdatingController and message to 0", func() {
+				It("should set rollback condition reason to UpdatingSegmentStore and message to 0", func() {
```

Comment:
Could you please add a test case for rollback with more than one segment store replicas.